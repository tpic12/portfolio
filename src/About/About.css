.about-section {
  width: 100%;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: flex-start;
  /* align-self: center; */
  /* background-color: rgb(168, 123, 72); */
  height: 100vh;
}

.about-circle {
  border-radius: 50%;
  width: 200px;
  height: 200px;
  border: 1px solid black;
  display: flex;
  justify-content: center;
  align-items: center;
  margin: 20px 0;
}

.about-me {
  height: 90vh;
  padding: 0 15px;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  color: rgb(62, 72, 117);
  background-color: white;
  margin-top: -1px;
  border-radius: 0 0 200px 200px;
}

.about-me p {
  padding-top: 20px;
}