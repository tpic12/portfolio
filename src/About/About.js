import React, { Component } from 'react';
import './About.css';

class About extends Component {
  render() {
    return (
      <section className='about-section snap'>
        <div className='about-me'>
            <p> &emsp; I am an NYC transplant from upstate New York, came here for a job on 
                Wall Street, enjoyed life behind a bar more, currently trying to get 
                out of the said bar. Now, a recent graduate from Thinkfulâ€™s Engineering Immersion, 
                I am looking to start a new career as a developer and professional 
                bug taker-outer (digital-kind only).

                <br/><br/>&emsp;I have always enjoyed solving problems, from how to add different 
                parts to my computer to rock climbing. Looking at something and figuring out 
                how to accomplish what looks impossible drives me to try, and then when it 
                gets done, to do it better. When a good challenge comes along I pride my self 
                on my adaptability, being in numerous work industries 
                (hospitality, to Wall Street, to Development) I have learned to take skills 
                learned and apply them in non-conventional ways, to view a problem from my 
                point of view, the Taylor v4.0 view. 
            </p>
        
        <div className='about-circle'>
            <span className='pic'>picture of me here</span>
        </div>
        </div>
      </section>
    );
  }
}

export default About;