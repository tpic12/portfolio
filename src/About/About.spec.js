import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import About from "./About";

describe("<Landing />", () => {
  it("renders by default", () => {
    const wrapper = shallow(<About />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
