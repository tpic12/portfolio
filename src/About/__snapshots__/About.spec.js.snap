// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> renders by default 1`] = `
<section
  className="about-section"
  id="about-section"
>
  <div
    className="about-me snap"
  >
    <p>
       
        A NYC transplant from upstate New York, I came here for a job on Wall Street, enjoyed life behind a bar more, and then decided said bar-life was no longer for me. Now, a recent graduate from Thinkful’s Engineering Immersion, I am looking to start a new career as a developer and professional bug taker-outer (digital-kind only).
      <br />
      <br />
       I have always enjoyed solving problems, from how to add different parts to my computer to rock climbing. Looking at something and figuring out how to accomplish what looks impossible drives me to try, and then when it gets done, to do it better. When a good challenge comes along, I pride my self on my adaptability, being in numerous work industries (hospitality, to Wall Street, to Development) I have learned to take skills acquired and apply them in non-conventional ways, to view a problem from my point of view, the Taylor v4.0 view.
    </p>
    <div
      className="about-circle"
    >
      <img
        alt="taylor"
        className="pic"
        src="headshot.png"
      />
    </div>
  </div>
</section>
`;
